convert.py

```diff
@@ -843,6 +911,9 @@ def add_meta_vocab(self, vocab: Vocab) -> None:
         self.gguf.add_token_scores(scores)
         self.gguf.add_token_types(toktypes)
```

Comment:
The signature of add_token_types needs to be changed to accept `List[ggml.TokenType]`.